test/docker.sh

```diff
@@ -29,6 +29,6 @@ export IMAGE=makeomatic/alpine-node:$NODE_VER
 $COMPOSE -f $DC up -d
 
 # make sure that services are up
-sleep 5
+sleep 25
```

Comment:
Ok, so we can implement it here, specifically, for testing, but for prod thats not really needed